### DISCUSSION TITLE
What is it for? (image)

### DISCUSSION THREAD

**Discussion by 7sorkr** [ponoor/step-series-support#22] [Q&A] [answered]
Can you tell me how the red part of the image is used?

![66351C56-1749-43CA-BE63-E90F325AFC36](https://user-images.githubusercontent.com/32606306/183350481-a4ae0c93-7407-4120-90d1-746d06b995a1.jpeg)

**Comment by kanta** [comment, marked as answer]
@7sorkr This is a SWD connector, a programing port for CORTEX-M processors. You can use dedicated programmers (like [this](https://www.microchip.com/en-us/development-tool/ATATMEL-ICE)) for burning a new firmware.
In STEP400 and STEP800, I burnt Arduino Zero bootloader from this port. Once the bootloader is burnt. then we can use USB port to upload the Arduino sketch.
